msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -295,7 +295,7 @@ class AccountState private constructor(
         LogSession.logMethodCall(
             tag = TAG,
             correlationId = null,
-            methodName = "$TAG.getAccessToken(forceRefresh: Boolean)"
+            methodName = "$TAG.getAccessToken(forceRefresh: Boolean = false)"
```

Comment:
Missing brackets -> {TAG}

Reply:
The correct methodName here should be getAccessTokenInternal(forceRefresh: Boolean, scopes: List<String>) right?

Also, maybe is better to print the actual value of the forceRefresh parameter to quickly identify if this was a forced token refresh by the application